docs/rules/README.md

```diff
@@ -8,266 +8,265 @@ Some rules are fixable using the `--fix` command line flag. Those rules are mark
+* [newline-per-chained-call](newline-per-chained-call.md): require a newline after each call in a method chain
+* [no-array-constructor](no-array-constructor.md): disallow `Array` constructors
+* [no-bitwise](no-bitwise.md): disallow bitwise operators
+* [no-continue](no-continue.md): disallow `continue` statements
+* [no-inline-comments](no-inline-comments.md): disallow inline comments after code
+* [no-lonely-if](no-lonely-if.md): disallow `if` statements as the only statement in `else` blocks
 * [no-mixed-spaces-and-tabs](no-mixed-spaces-and-tabs.md): disallow mixed spaces and tabs for indentation (recommended)
 * [no-multiple-empty-lines](no-multiple-empty-lines.md): disallow multiple empty lines
 * [no-negated-condition](no-negated-condition.md): disallow negated conditions
 * [no-nested-ternary](no-nested-ternary.md): disallow nested ternary expressions
-* [no-new-object](no-new-object.md): disallow the use of the `Object` constructor
-* [no-plusplus](no-plusplus.md): disallow use of unary operators, `++` and `--`
-* [no-restricted-syntax](no-restricted-syntax.md): disallow use of certain syntax in code
-* [no-spaced-func](no-spaced-func.md): disallow space between function identifier and application (fixable)
-* [no-ternary](no-ternary.md): disallow the use of ternary operators
+* [no-new-object](no-new-object.md): disallow `Object` constructors
+* [no-plusplus](no-plusplus.md): disallow the unary operators `++` and `--`
+* [no-restricted-syntax](no-restricted-syntax.md): disallow specified syntax
+* [no-spaced-func](no-spaced-func.md): disallow spacing between function identifiers and their applications (fixable)
+* [no-ternary](no-ternary.md): disallow ternary operators
 * [no-trailing-spaces](no-trailing-spaces.md): disallow trailing whitespace at the end of lines (fixable)
 * [no-underscore-dangle](no-underscore-dangle.md): disallow dangling underscores in identifiers
-* [no-unneeded-ternary](no-unneeded-ternary.md): disallow the use of ternary operators when a simpler alternative exists
+* [no-unneeded-ternary](no-unneeded-ternary.md): disallow ternary operators when simpler alternatives exist
 * [no-whitespace-before-property](no-whitespace-before-property.md): disallow whitespace before properties
 * [object-curly-spacing](object-curly-spacing.md): require or disallow padding inside curly braces (fixable)
-* [one-var](one-var.md): require or disallow one variable declaration per function
-* [one-var-declaration-per-line](one-var-declaration-per-line.md): require or disallow an newline around variable declarations
-* [operator-assignment](operator-assignment.md): require assignment operator shorthand where possible or prohibit it entirely
-* [operator-linebreak](operator-linebreak.md): enforce operators to be placed before or after line breaks
-* [padded-blocks](padded-blocks.md): enforce padding within blocks
+* [one-var](one-var.md): enforce variables to be declared at the same time or separately per function
```

Comment:
declared either together or separately

You might think of an even clearer word to communicate the contrast.